### ISSUE TITLE
all IANA cipher suites

### ISSUE THREAD

**Issue by kylak** [jvehent/cipherscan#193]
Hello, does cipherscan uses all IANA cipher suites ?

Thanks.

**Comment by tomato42** [comment]
No, it doesn't. It can detect only the ones that the included version of openssl knows about.

#152 is about fixing it (making the python code perform the check, not openssl command line utility)

**Comment by kylak** [comment]
Ok, thanks for the information.
